### PR TITLE
DOC Deprecate CwpSiteSummaryExtension

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/cwp/issues/320

### CODE REVIEW

File: en/04_Changelogs/4.13.0.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ title: 4.13.0 (unreleased)
 
 The following API have been deprecated in this minor release, and will be removed in a future major release. For API with a replacement, we recommend you update your code to use the replacement as part of your upgrade to Silverstripe CMS 4.13.0.
 
+#### cwp/cwp
+
+Deprecated class [`CWP\CWP\Extensions\CwpSiteSummaryExtension`](api:CWP\CWP\Extensions\CwpSiteSummaryExtension) Will be removed without equivalent functionality to replace it
```
Comment (ChloeSartorelli) [inline en/04_Changelogs/4.13.0.md]:
```suggestion
- Deprecated class [`CWP\CWP\Extensions\CwpSiteSummaryExtension`](api:CWP\CWP\Extensions\CwpSiteSummaryExtension) Will be removed without equivalent functionality to replace it
```
I'm converting these to lists as part of the beta release process (they should have been lists to start with - oops). This PR has missed the window for being part of that conversion so please update in PR.

Comment (ChloeSartorelli) [inline en/04_Changelogs/4.13.0.md]:
```suggestion
Deprecated class `CWP\CWP\Extensions\CwpSiteSummaryExtension` Will be removed without equivalent functionality to replace it
```
Not in API docs: https://api.silverstripe.org/4/search.html?search=CwpSiteSummaryExtension

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/04_Changelogs/4.13.0.md (hunk 1 above)
Comment (emteknetnz) [reply en/04_Changelogs/4.13.0.md]: Updated

Comment (emteknetnz) [reply en/04_Changelogs/4.13.0.md]: Made an issue instead https://github.com/silverstripe/.github/issues/24

**Review by GuySartorelli — APPROVED** [review]

**Comment by GuySartorelli** [comment]
Retargetted to `4.13` now that that branch exists. Conflicts to resolve.

**Comment by emteknetnz** [comment]
Have rebased

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**